### ISSUE TITLE
Implement SVGGeometryElement

### ISSUE THREAD

**Issue by matthelliwell2** [svgdotjs/svgdom#109]
I need to test my calls various calls to the [SVGGeometryElement](https://developer.mozilla.org/en-US/docs/Web/API/SVGGeometryElement) methods, eg isPointInFill. It doesn't look like svgdom implements these methods. Are there any plans to implement them?

**Comment by Fuzzyma** [comment]
No, not yet. This library grows with it's requirements so feel free to implement the methods you need :).

pointInFill sounds like a bigger undertaking tho especially for complex shapes

**Comment by matthelliwell2** [comment]
I did have a look at this but it's beyond me once you start with stroke widths and ends. Back to Karma is it I suppose.

**Comment by Fuzzyma** [comment]
well, isPointInFill does NOT include strokes. That would be **way** harder :D
If you need isPointInStroke - well, yes, thats an undertaking on its own
